### PR TITLE
Location admin overview

### PR DESCRIPTION
**Pull request by jasperweyne**
This PR adds an overview over all locations stored in the database. It's intended as a first step towards implementing #257 in a sane way: when an activity has occurred in a location earlier, this change allows a user to view the evaluations not only for an earlier activity, but also multiple evaluations for the same location. It also implements #312, by adding location information to the activity admin overview and integrating this with this location overview through a clickable link.

Given the (previous) 1-to-1 relation from Activities to Locations, a lot of duplicate Locations will probably have occurred in production databases. The simplicity of the Location entity makes this a good testing ground for implementing the merging of database entries with foreign key relations present.

### CODE REVIEW

File: src/Entity/Location/Location.php [hunk 1]
```diff
@@ -54,4 +70,37 @@ public function setAddress(string $address): self
...
+     */
+    public function getActivities(): Collection
+    {
+        return $this->activities;
+    }
+
+    public function addActivity(Activity $activity): self
+    {
+        if (!$this->activities->contains($activity)) {
+            $this->activities[] = $activity;
```
Comment (DavidckPixel) [inline src/Entity/Location/Location.php:87]: Maybe an additional check here that the activity also has this location associated with it?

**Review by DavidckPixel — CHANGES_REQUESTED** [review]
There is still a bug within the system where the Location Menu did not show up in the side bar of the admin. Github marked that bug. 

Overall it looks good.

However with the current implementation of location when making a new activity with the exact same location in name, it still creates a new location entity, and there is no easy way to set an existing activities locations to an already existing entity. This causes the system to constantly create new Locations with the same name. It would be nice to have a way to select from an already existing location when creating a new activity.

File: src/Entity/Location/Location.php (hunk 1 above)
Comment (A-Daneel) [reply src/Entity/Location/Location.php:82]: We'll address this in a separate PR

**Review by A-Daneel — APPROVED** [review]
re-merged develop and resolved the linting issues. merge and move on!

**Pull request merged**